### ISSUE TITLE
Set up no longer works

### ISSUE THREAD

**Issue by elie222** [saxenashivang/cypress-metamask-v2#8]
The automatic MetaMask setup no longer works. It gets stuck at the enter secret key screen.

I believe the issue is that the selector for the private key field has changed:

Current input:

```html
<input aria-invalid="false" placeholder="Paste Secret Recovery Phrase from clipboard" type="password" dir="auto" class="MuiInputBase-input MuiInput-input" value="">
```

Code to enter secret:

```js
const secretWordsInput = `${firstTimeFlowFormPage} .import-srp__srp-word input[data-testid="import-srp__srp-word-%"]`;
```

**Comment by fabriciomirandabr** [comment]
I just discovered this package right now, and I'm facing the same problem, the password input by cypress are not the same that I'm using on .env

**Comment by alam5** [comment]
I used this library for the first time. After configuring it and running it for first time I faced this error related to `setupMetamask()` function. 
![runfirst](https://user-images.githubusercontent.com/10685678/179828104-e7f963d1-a8d4-4a02-8986-025b47e2feaf.png)

**Comment by elie222** [comment]
Yeah. Quite a lot seems to be broken. Even running the e2e within the package itself fails if you do:

```sh
npm i
npm run start
npm run cy:run
```

**Comment by ledahu05** [comment]
I started to use the package today. First of all, despite the issues listed above, I think it's working great - so thank you.

Nevertheless the **automatic** setup of metamask (1/ paraphrase, password and network 2/login to metamask with pasword) is not working for me. I need to do the manual setup once and to login to metamask everytime. But for the tests themselves it's working great.  Since we are talking about automation it would be great if it was possible.

Thanks.

**Comment by dlpigpen** [comment]
Got the same issue, It gets stuck at the enter secret key screen.

**Comment by rubelux** [comment]
I need help to make this work.

**Comment by elie222** [comment]
From when I used it it seems like the set up code was written for an old MetaMask UI and needs a full update to start working again.

Best,
Eliezer
